packages/abi-coder/src/fragments/param-type.ts

```diff
@@ -0,0 +1,175 @@
+import { Logger } from '@ethersproject/logger';
+import { defineReadOnly } from '@ethersproject/properties';
+
+import { arrayRegEx, enumRegEx, structRegEx } from '../abi-coder';
+
+const version = 'abi/5.6.4';
```

Comment:
The version here is the version of the current package; in this case, `v0.10.0` you could query from the `package.json`.

Reply:
I used as I see other places in the repo use this same logger pattern.
`const logger = new Logger(process.env.BUILD_VERSION || '~');`,